res/linux/org.mixxx.Mixxx.metainfo.xml

```diff
@@ -1106,7 +1106,7 @@
  </ul>
 </description>
     </release>
-    <release version="2.3.4" type="development">
+    <release version="2.3.4" type="development" date="2022-08-05" timestamp="1659651020">
```

Comment:
I guess an unintended side effect is now that the releases always get updated to the newest timestamp, ignoring which parts of the changelog were actually changed in that commit...

Reply:
Not really unindented, I was aware of that. There's just a point where the additional complexity does not justify the work. The timestamp for dev versions is wrong anyway - it doesn't get update when a regular commit is made that doesn't change the changelog but changes the actual code. Otherwise every single commit would requires changes to the metainfo file.

See https://github.com/mixxxdj/mixxx/pull/4880/files#diff-98777318ab7af5cd0c403ac9da4897de950a6ad4947a0acfc78192c82305d018R84-R92 for details.

Reply:
Right, I missed the sentence `The same goes for using the latest commit date.`

I guess its fine as is for now. Though in the long term moving the metainfo out of the pre-commit hook into a packaging step is probably better...

Reply:
Yes. I think the reason for keeping it here was to use the same metadata for flatpak, and also to avoid introducing python as a build dependency. But I think it may be too much of a compromise.

Reply:
I think introducing python as a linux-only build dependency is probably not that big of a deal.